internal/backends/winit/renderer/femtovg/glcontext.rs

```diff
@@ -69,62 +69,70 @@ impl OpenGLContext {
                 }
             }
 
-            let try_create_surface = |display_api_preference| -> glutin::error::Result<(_, _)> {
-                let gl_display = unsafe {
-                    glutin::display::Display::new(
-                        _display.raw_display_handle(),
-                        display_api_preference,
-                    )?
+            let width: std::num::NonZeroU32 =
+                _size.width.try_into().expect("new context called with zero width window");
+            let height: std::num::NonZeroU32 =
+                _size.height.try_into().expect("new context called with zero height window");
```

Comment:
This is ah expect for now, but as part of the other report I'll try to replace these with return-on-error paths. For now I just want to know what's causing the panic in the previewer.